.github/dependabot.yml

```diff
@@ -1,10 +1,14 @@
 version: 2
 updates:
   - package-ecosystem: "github-actions"
-    directory: "/" # Location of package manifests
+    directory: "/"
     schedule:
       interval: "weekly"
```

Comment:
would this be eligible for grouping rules too?

Reply:
Yes, I meant to. Updated in b2e5505